modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -500,7 +503,8 @@ public void testGetSearchContainerWithDefaultOrder() throws Exception {
 		SearchContainer<Object> searchContainer = _getSearchContainer(
 			_getMockLiferayPortletRenderRequest());
 
-		Assert.assertEquals(2, searchContainer.getTotal());
+		Assert.assertEquals(
+			defaultSearchContainer.getTotal() + 2, searchContainer.getTotal());
```

Comment:
Could we obtain defaultSearchContainer value, without calling the searchContainer, calling directly to a DLFileEntry method?

Reply:
I've been looking at this and I am not sure that we could do it differently (easily). As to make it work in a general way, what we need to do is retrieve everything that is already there (currently is just a doc but in the future could be something else (or less) and I'd like this test to be flexible enough to be able to manage that)

What do you think about the `getCompanyEntriesCount(..)` method in the AssetEntryService? Not sure if this is too elaborated just for a test or maybe is not accurate enough
Let me know your thoughts :)

Reply:
The problem is that we not only support Asset, so we can keep it that way :)